ledger/ledger-api-test-tool-on-canton/BUILD.bazel

```diff
@@ -189,7 +189,6 @@ conformance_test(
     test_tool_args = [
         "--verbose",
         "--concurrent-test-runs=1",  # lowered from default #procs to reduce flakes - details in https://github.com/digital-asset/daml/issues/7316
-        "--timeout-scale-factor=4",  # provide additional time to avoid timing on on longer tests such as PRLocalAndNonLocalRetroactiveDivulgences or PRImmediateAndRetroactiveDivulgence
```

Comment:
@oliverse-da Removed it here, since the tests have the higher timeout scale factor built-in